src/static/alchemy/alchemy_config.js

```diff
@@ -962,6 +991,12 @@ var alchemy_config = {
             "width": 1,
             "opacity": 0.8,
             "color": "#0000FF"
+        },
+        "v2_label_ref": {
+            "width": 1,
+            "opacity": 0.3,
+            //"color": "#0000FF"
```

Comment:
Is this a necessary comment?

Reply:
Thanks your comment.
This is unnecessary. I will remove it.